ykcompile/src/lib.rs

```diff
@@ -34,10 +34,15 @@ lazy_static! {
     static ref CALLER_SAVE_REGS: [u8; 8] = [RDI.code(), RSI.code(), RDX.code(), RCX.code(),
                                             R8.code(), R9.code(), R10.code(), R11.code()];
 
+    // Registers that are callee-save as per the Sys-V ABI.
+    // Note that RBP is also callee-save, but is handled specially.
+    static ref CALLEE_SAVE_REGS: [u8; 5] = [RBX.code(), R12.code(), R13.code(),
```

Comment:
Should we rename this to `callee_saved_regs` to avoid confusion. Someone might think this means `callee-safe` (as in these registers are safe for the callee to use a.k.a caller-saved).

Reply:
Seems sensible!